account_wallet/migrations/14.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2022-Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.column_exists(env.cr, "account_wallet_type", "no_anonymous"):
+        openupgrade.rename_columns(
```

Comment:
TypeError: rename_columns() missing 1 required positional argument: 'column_spec'

should be 

```suggestion
        openupgrade.rename_columns(env.cr, ...
```